packages/main/cypress/specs/base/Events.cy.ts

```diff
@@ -156,12 +176,26 @@ describe("Event bubbling", () => {
 
 		cy.get("@dialog")
 			.then(dialog => {
+				dialog.get(0).addEventListener("close", cy.stub().as("dialogClosed"));
```

Comment:
I'm not sure about naming the aliases and when they are linked to two different stubs. Can we rename one of them to dialogClosed2?